test-build.sh

```diff
@@ -29,9 +29,19 @@ for version in "${versions[@]}"; do
 
   OUTPUT=$(docker run --rm -it node:"$tag" node -e "process.stdout.write(process.versions.node)")
   if [ "$OUTPUT" != "$full_version" ]; then
-    fatal "Test of $tag failed!"
+    fatal "Test of $tag for node failed!"
   fi
-  info "Test of $tag succeeded."
+  info "Test of $tag for node succeeded."
+
+  if ! docker run --rm -it node:"$tag" npm --version; then
+    fatal "Test of $tag for npm failed!"
+  fi
+  info "Test of $tag for npm succeeded."
+
+  if ! docker run --rm -it node:"$tag" yarn --version; then
```

Comment:
Nice!

Reply:
thx! maybe the -t is completely useless here...